### PR TITLE
SRB2 auto fullscreen resolution

### PR DESCRIPTION
**Pull request by theofficialgman**
changed default renderer to opengl, and added resolution detection at boot so game fullscreen works nativly in both docked and portable mode without user intervention

### CODE REVIEW

**Comment by cobalt2727** [comment]
I noticed the script assumes gnome-terminal is the terminal app - maybe later on we can somehow change this to not rely on it in case someone's using a different DE and removed the GNOME terminal for whatever reason...

**Comment by cobalt2727** [comment]
Well, either that or we just make it sudo apt install the GNOME terminal just in case, but that's a bit of a messier solution

**Comment by theofficialgman** [comment]
yeah, however the .desktop file executes files is not the same as how they are executed in terminal so I had to do it that way. The script just wouldn't execute otherwise. If anyone else finds a more elegant solution that works go ahead and implement it.

**Comment by theofficialgman** [comment]
TLDR: executing the .sh file worked fine in terminal but crashed .desktop
